File: src/main/java/Calculator.java
public class Calculator {
    public int calculate(int val1, String operator, int val2) {

        var result = switch (operator) {
            case "+" -> val1 + val2;
            case "-" -> val1 - val2;
            case "/" -> val1 / val2;
            case "*" -> val1 * val2;
            default -> 0;
        };


        return result;
    }
}

File: hw2/src/main/kotlin/Calculator.kt
import java.lang.ArithmeticException

open class Calculator {
    fun calculate(val1: Int, operator: String, val2: Int) = when(operator){
                "+" -> val1 + val2
                "-" -> val1 - val2
                "*" -> val1 * val2
                "/" -> {
                    if (val2 != 0)
                        val1 / val2
                    else throw ArithmeticException("You can`t divide by zero!")
                }
                else -> throw UnsupportedOperationException("Operator is not supported!")
            }
    }
